lib/irc.js

```diff
@@ -889,6 +847,100 @@ Client.prototype.ctcp = function(to, type, text) {
     return this[type === 'privmsg' ? 'say' : 'notice'](to, '\1'+text+'\1');
 }
 
+function Channel(name, users, mode){
+    this.key = name.toLowerCase();
+    this.serverName = name;
+    this.users = users || {};
+    this.mode = mode || '';
+    this.setting = {};
+}
+exports.Channel = Channel;
+Channel.prototype.key = '';
```

Comment:
I'm just hinting at the type that each property will be once the constructor has finished its inititialization. It might be unnecessary for most purposes, and it definitely should't make a difference during execution.